core/lib/spree/permission_sets/user_management.rb

```diff
@@ -2,7 +2,15 @@ module Spree
   module PermissionSets
     class UserManagement < PermissionSets::Base
       def activate!
-        can :manage, Spree.user_class
+        # due to how cancancan filters by associations,
+        # we have to define this twice, once for `accessible_by`
+        can :manage, Spree.user_class, spree_roles: { id: nil }
+        # and once for `can?`
+        can :manage, Spree.user_class do |user|
```

Comment:
Nope. This happens at runtime, so `Spree.user_class` is fine. `Spree::UserClassHandle` is only needed if used during initialization.